polaris-react/src/components/Page/components/Header/components/Title/Title.module.scss

```diff
@@ -33,9 +33,12 @@
 .SubTitle {
   margin-top: var(--p-space-050);
   color: var(--p-color-text-secondary);
-  max-width: 45ch;
 
   &.SubtitleCompact {
     margin-top: var(--p-space-050);
   }
+
+  &.SubtitleMaxWidth {
+    max-width: 45ch;
```

Comment:
@svintit It's so that a long subtitle wouldn't always mean that every single secondary action / action group was rolled up behind a "More actions" button. secondaryActions and actionGroups can contain actions that merchants make frequently, and forcing the merchant to always make an additional click to open the popover if they're always rolled up isn't particularly desirable, if we can ensure that the subtitle wraps and we have enough space to render the first few buttons on the page, rather than behind the popover.

I've attached screenshots of how it can look if we don't auto wrap the subtitle at a certain width, where you can see how busy the popover can get.


<img width="1624" alt="Screenshot 2024-02-14 at 16 01 27" src="https://github.com/Shopify/polaris/assets/2562596/4e6a6f91-9449-4e59-89d8-ea96742a530a">

<img width="1624" alt="Screenshot 2024-02-14 at 16 01 31" src="https://github.com/Shopify/polaris/assets/2562596/4fd8b04d-3f20-4d76-8e2d-e5fd18ee1a67">

Reply:
It's a _somewhat_ magic number based off data for the optimal line length of text, that also allows for a decent amount of room to render the actions to the right-hand-side. It's never going to be a one-size-fits-all type solution, but we went with what _should_ fit _most_ cases

Reply:
Makes sense, LGTM! Thanks for doing this!